### DISCUSSION TITLE
Is it possible to wrap long lines of code in a PDF?

### DISCUSSION THREAD

**Discussion by magical-heyrovsky** [quarto-dev/quarto-cli#5728] [Q&A] [answered]
### Description

Let's say I have a document like this:

````qmd
---
format:
  pdf:
    colorlinks: true
---

```python
def main():
        # Do some stuff...
        response = requests.request("GET", "https://codeberg.org/api/v1/repos/search", data="", params=querystring)
        # Do more stuff...
```
````

Is there any way I can prevent the PDF from looking like this without manually changing the code structure? I saw it's possible with HTML: https://quarto.org/docs/output-formats/html-code.html#code-overflow but is there anything like this for PDF?

![image](https://github.com/quarto-dev/quarto-cli/assets/101060148/e9d63aa4-9bdc-4eb5-93f3-da768e9375e8)

Quarto version: 1.3.333
OS: Ubuntu 23.04

**Comment by mcanouil** [comment, marked as answer]
Closing this as duplicate of #4121 (and #3693)
